packages/sitecore-jss/src/personalization/layout-personalization-service.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  ComponentRendering,
+  PersonalizedComponentRendering,
+  LayoutServiceContext,
+  RouteData
+} from '../layout/models';
+import {
+  PersonalizationDecisionsService,
+} from './personalization-decisions-service';
+import {
+  LayoutFragmentService,
+  LayoutFragmentData,
+} from './layout-fragment-service';
+import {
+  LayoutPersonalizationUtils
+} from './layout-personalization-utils'
+
+export interface PersonalizationResult {
+  route?: RouteData;
+  isRoutePersonalized: boolean;
+}
+
+export interface PersonalizationLoadResult {
+  hasPersonalizationComponents: boolean;
+  personalizedFragments?: { [key: string]: ComponentRendering | null }
+}
+
+export class LayoutPersonalizationService {
```

Comment:
All fields and functions of LayoutPersonalizationService are public by default.
Is it expected?

Reply:
marked as private